lib/fizzy/execute.cpp

```diff
@@ -340,12 +342,21 @@ inline bool load_from_memory(
     return true;
 }
 
+template <typename T>
+inline T pop_store_memory_arg(OperandStack& stack) noexcept
```

Comment:
I don't see reason to pass `OperandStack` in. Better to take `Value` as argument and make it similar to `extend`. Maybe call it `shrink` or `trunc` then?